### ISSUE TITLE
[Bug]: Class "App\Livewire\Components\Section" not found

### ISSUE THREAD

**Issue by 0xf965** [rupadana/filament-swiper#4]
### What happened?

Using the component throws this error:

![Captura desde 2024-01-24 10-44-02](https://github.com/rupadana/filament-swiper/assets/40121100/6f3c5acc-226b-45d2-acc3-a8392f9bd075)

What happen?

### How to reproduce the bug

Use the example on a Resource's Infolist.

 I installed this plugin with the following libraries:

```json
{
    "php": "^8.1",
    "andreiio/blade-remix-icon": "^2.5",
    "filament/filament": "^3.1-stable",
    "flowframe/laravel-trend": "^0.1.5",
    "guzzlehttp/guzzle": "^7.2",
    "itsgoingd/clockwork": "^5.1",
    "laravel-notification-channels/fcm": "^3.2",
    "laravel/framework": "^10.10",
    "laravel/sanctum": "^3.2",
    "laravel/tinker": "^2.8",
    "pxlrbt/filament-excel": "^2.2",
    "rupadana/filament-swiper": "^3.0@beta",
    "saade/filament-fullcalendar": "^3.0",
    "spatie/laravel-fractal": "^6.0"
}
```

I have tried using the example code:

```php
SwiperImageEntry::make('captures.url')
    ->navigation(false)
    ->pagination()
    ->paginationType(SwiperImageEntry::BULLETS)
    ->paginationClickable()
    ->paginationDynamicBullets()
    ->paginationHideOnClick()
    ->paginationDynamicMainBullets(2)
    ->scrollbar()
    ->scrollbarDragSize(100)
    ->scrollbarDraggable()
    ->scrollbarSnapOnRelease()
    ->scrollbarHide(false)
    ->height(300)
    ->autoplay()
    ->effect(SwiperImageEntry::CARDS_EFFECT)
    ->cardsPerSlideOffset(2)
    ->autoplayDelay(500)
    ->centeredSlides()
    ->slidesPerView(2);
```

### Package Version

3.0.0-beta

### PHP Version

8.1.26

### Laravel Version

10.10

### Which operating systems does with happen with?

Linux

**Comment by eele94** [comment]
hotfix for now:
Create app/Livewire/Section.php

```
<?php

namespace App\Livewire\Components;

use Closure;
use Filament\Infolists\Components\Component;
use Filament\Infolists\Components\Concerns\CanBeCollapsed;
use Filament\Infolists\Components\Concerns\CanBeCompacted;
use Filament\Infolists\Components\Concerns\EntanglesStateWithSingularRelationship;
use Filament\Infolists\Components\Concerns\HasHeaderActions;
use Filament\Infolists\Components\Contracts\HasHeaderActions as ContractsHasHeaderActions;
use Filament\Support\Concerns\HasDescription;
use Filament\Support\Concerns\HasExtraAlpineAttributes;
use Filament\Support\Concerns\HasHeading;
use Filament\Support\Concerns\HasIcon;
use Filament\Support\Concerns\HasIconColor;
use Illuminate\Contracts\Support\Htmlable;
use Illuminate\Support\Str;

class Section extends Component implements ContractsHasHeaderActions
{
    use CanBeCollapsed;
    use CanBeCompacted;
    use EntanglesStateWithSingularRelationship;
    use HasDescription;
    use HasExtraAlpineAttributes;
    use HasHeaderActions;
    use HasHeading;
    use HasIcon;
    use HasIconColor;

    /**
     * @var view-string
     */
    protected string $view = 'filament-infolists::components.section';

    protected bool | Closure | null $isAside = null;

    protected bool | Closure $isContentBefore = false;

    /**
     * @param  string | array<Component> | Htmlable | Closure | null  $heading
     */
    final public function __construct(string | array | Htmlable | Closure | null $heading = null)
    {
        is_array($heading)
            ? $this->childComponents($heading)
            : $this->heading($heading);
    }

    /**
     * @param  string | array<Component> | Htmlable | Closure | null  $heading
     */
    public static function make(string | array | Htmlable | Closure | null $heading = null): static
    {
        $static = app(static::class, ['heading' => $heading]);
        $static->configure();

        return $static;
    }

    protected function setUp(): void
    {
        parent::setUp();

        $this->columnSpan('full');
    }

    public function aside(bool | Closure | null $condition = true): static
    {
        $this->isAside = $condition;

        return $this;
    }

    public function getId(): ?string
    {
        $id = parent::getId();

        if (filled($id)) {
            return $id;
        }

        $heading = $this->getHeading();

        if (blank($heading)) {
            return null;
        }

        $id = Str::slug($heading);

        if ($statePath = $this->getStatePath()) {
            $id = "{$statePath}.{$id}";
        }

        return $id;
    }

    public function getKey(): ?string
    {
        return parent::getKey() ?? ($this->getActions() ? $this->getId() : null);
    }

    public function isAside(): bool
    {
        return (bool) ($this->evaluate($this->isAside) ?? false);
    }

    public function contentBefore(bool | Closure $condition = true): static
    {
        $this->isContentBefore = $condition;

        return $this;
    }

    public function isContentBefore(): bool
    {
        return (bool) $this->evaluate($this->isContentBefore);
    }
}
```

**Comment by rupadana** [comment]
Thanks for the report

Fixed on version [3.0.1-beta](https://github.com/rupadana/filament-swiper/tree/3.0.1-beta)
